src/core/cookie.js

```diff
@@ -77,15 +77,14 @@ DM.provide('Cookie',
      */
     load: function()
     {
-        // note, we have the opening quote for the value in the regex, but do
-        // not have a closing quote. this is because the \b already handles it.
-        var cookie = decodeURIComponent(document.cookie).match('\\bdms_' + DM._apiKey + '="([^;]*)\\b'),
-            session;
+        var session = undefined;
+        var key = 'dms_' + DM._apiKey;
+        var cookie = Cookies.get(key);
```

Comment:
The DM SDK should not _polute_ the global scope with anything but the `DM` namespace. You must find a way to include `Cookies` export into the `DM` namespace.